src/EditorFeatures/Test2/ReferenceHighlighting/VisualBasicReferenceHighlightingTests.vb

```diff
@@ -272,5 +272,27 @@ End Class
 
             VerifyHighlights(input)
         End Sub
+
+        <WorkItem(1904, "https://github.com/dotnet/roslyn/issues/1904")>
+        <WorkItem(2079, "https://github.com/dotnet/roslyn/issues/2079")>
```

Comment:
@dpoeschl and I just had a conversation about this and it ultimately comes down to people being in one of two camps:

1.  The attribute should be added to note a relationship, sort of like a cref.
2.  The attribute should be added once the bug is fixed.

David's in camp 1, I'm in 2, but ultimately I don't care really strongly about this either way and I should have prefaced my comment with "nit".

Back on the real issue of your fix, :shipit: